build.gradle

```diff
@@ -50,10 +44,9 @@ buildscript {
   }
 
   dependencies {
-    classpath dep.androidPlugin
     classpath dep.kotlinGradlePlugin
-    classpath dep.dokkaAndroid
     classpath dep.dokka
+    classpath dep.androidPlugin
     classpath 'de.undercouch:gradle-download-task:3.1.2'
```

Comment:
I know it is not apart of this PR but could you put `de.undercouch:gradle-download-task:3.1.2` in `dep`?